Dockerfile

```diff
@@ -24,9 +30,9 @@ RUN microdnf install shadow-utils yum
 RUN useradd -ms /bin/bash rhoas
 USER rhoas
 
-COPY contrib/rhoas /usr/local/bin/rhoas
 COPY contrib/oc /usr/local/bin/oc
```

Comment:
Do you plan adding scripts in that PR? It will be easier as we can test image with scripts while testing this PR
See #10

Reply:
Was going to do it in a separate PR, but happy to do it in here.